rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -85,7 +87,9 @@ def apply_rough_alignment(render,
         # get the lowres stack rough alignment transformation
         tforms = lowres_ts[0].tforms
         tf = tforms[-1]
-        tf.M[0:2,0:2]*=scale
+        
+        if apply_scale:
+            tf.M[0:2,0:2]*=scale
```

Comment:
I think this should have an else, doing something like 
```python
tf.M[:2, -1] /= scale
```
Which will account for the scaling of the solution's translation (unnecessary when scale is applied to the downsamples).

Reply:
The else would not be required if the scaling is not applied to downsamples... which will be the case from now on in the workflow... I am not sure I am following the "scaling of the solution's translation" here..